lib/mpl_toolkits/mplot3d/tests/test_axes3d.py

```diff
@@ -2124,3 +2124,105 @@ def test_panecolor_rcparams():
                          'axes3d.zaxis.panecolor': 'b'}):
         fig = plt.figure(figsize=(1, 1))
         fig.add_subplot(projection='3d')
+
+
+def test_mutating_input_arrays_y_and_z():
+    fig = plt.figure()
+    ax = fig.add_subplot(111, projection='3d')
+    x = [1, 2, 3]
+    y = [0.0, 0.0, 0.0]
+    z = [0.0, 0.0, 0.0]
+    pl, = ax.plot(x, y, z, 'o-')
+
+    ax.set_ylim([0, 4])
+    ax.set_zlim([0, 4])
+
+    # mutate y,z to get a nontrivial line
+    y[:] = [1, 2, 3]
+    z[:] = [1, 2, 3]
+
+    fig.draw_without_rendering()
+    pl.get_data_3d()
+    pl.get_data()
+
+    array = tuple
+    result1 = (array([1, 2, 3]), array([0., 0., 0.]), array([0., 0., 0.]))
```

Comment:
```suggestion
    result1 = ((1, 2, 3),(0., 0., 0.), (0., 0., 0.))
```
allclose will check that the numerical values are close, so if using lists or tuples (or np.array) doesn't matter as long as the structure is correct. (Modulo that it may not be correct in my simplified version...)

Same on the next row.